src/elements/common/annotator-context/types.ts

```diff
@@ -7,10 +7,10 @@ export enum Action {
 }
 
 export interface AnnotatorState {
-    activeAnnotationId?: string;
-    annotation?: object | undefined;
-    action: Action | undefined;
-    error: Error | undefined;
+    activeAnnotationId?: string | null;
```

Comment:
Will removing the null here and all the null in withAnnotations.tsx work?

Reply:
We want `null` though because that's a valid assignment for `activeAnnotationId` which represents that there is no active annotation id